src/CartItem.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import App from './App.js';
+import InventoryProd from './InventoryProd.js';
+
+class CartItem extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            //props: props
+        }
+    }
+
+    addProduct = event => {
+        this.props.changeCart(this.props.el);
+    }
+
+    minusProduct = event => {
+        console.log('minus ', event.target);
+    }
+
+    render() {
+        return(
+            <div className="Single-product" id={this.props.el.inventory_code}>
```

Comment:
I'm seeing a lot of `div` and `span` elements in the JSX... I would err on the side of changing these into semantic HTML elements now while it's still early in your development. For reasons of accessibility, SEO, and dev empathy - `div` and `span` should only be used as a [last resort](https://www.w3.org/TR/html51/grouping-content.html#the-div-element).